airflow/providers/amazon/aws/hooks/ecs.py

```diff
@@ -185,7 +185,15 @@ def _get_log_events(self, skip: int = 0) -> Generator:
         except ClientError as error:
             if error.response["Error"]["Code"] != "ResourceNotFoundException":
                 self.logger.warning("Error on retrieving Cloudwatch log events", error)
-
+            else:
+                self.logger.info(
+                    "Cannot find log stream yet, it can take a couple of seconds to show up.\n"
+                    "If this error persists, check that the log group and stream are correct:\n"
+                    "group: %s\n"
+                    "stream: %s",
```

Comment:
Any specific reason this log should be multiple lines? A log message with literal newlines is generally avoided since it breaks log formatting (timestamp prefixes, for example). If the lines are intended to separate the situation summary and details, it is more preferred to log them separately instead.